tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -551,6 +564,187 @@ describe(__filename, () => {
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_FIREFOX,
+        name: 'Alerts & Update',
+        slug: 'alert-update',
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_FIREFOX,
+        name: 'Security',
+        slug: 'security',
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_FIREFOX,
+        name: 'Anime',
+        slug: 'anime',
+        type: ADDON_TYPE_STATIC_THEME,
+      },
+    ];
+
+    it('renders related categories', () => {
+      const addon = createInternalAddonWithLang({
+        ...fakeAddon,
+        categories: {
+          [CLIENT_APP_FIREFOX]: ['alert-update', 'security'],
+        },
+        type: ADDON_TYPE_EXTENSION,
+      });
+
+      store.dispatch(loadCategories({ results: categories }));
+      store.dispatch(setClientApp(CLIENT_APP_FIREFOX));
+
+      const root = render({ addon, store });
+
+      expect(
+        root.find('.AddonMoreInfo-related-categories').find(Link),
+      ).toHaveLength(2);
+      expect(
+        root
+          .find('.AddonMoreInfo-related-categories')
+          .find(Link)
+          .at(0)
+          .prop('to'),
+      ).toEqual('/extensions/category/alert-update/');
+      expect(
+        root
+          .find('.AddonMoreInfo-related-categories')
+          .find(Link)
+          .at(1)
+          .prop('to'),
+      ).toEqual('/extensions/category/security/');
+    });
+
+    it('does not render related categories when add-on has no category', () => {
+      const addon = createInternalAddonWithLang({
+        ...fakeAddon,
+        categories: {
+          [CLIENT_APP_FIREFOX]: [],
+        },
+        type: ADDON_TYPE_EXTENSION,
+      });
+
+      store.dispatch(loadCategories({ results: categories }));
+      store.dispatch(setClientApp(CLIENT_APP_FIREFOX));
+
+      const root = render({ addon, store });
+
+      expect(
+        root.find('.AddonMoreInfo-related-categories').find(Link),
+      ).toHaveLength(0);
+    });
+
+    it('does not render related categories when there are no loaded categories', () => {
+      const addon = createInternalAddonWithLang({
+        ...fakeAddon,
+        categories: {
+          [CLIENT_APP_FIREFOX]: ['alert-update', 'security'],
+        },
+        type: ADDON_TYPE_EXTENSION,
+      });
+
+      store.dispatch(loadCategories({ results: [] }));
+      store.dispatch(setClientApp(CLIENT_APP_FIREFOX));
+
+      const root = render({ addon, store });
+
+      expect(
+        root.find('.AddonMoreInfo-related-categories').find(Link),
+      ).toHaveLength(0);
+    });
+
+    it('does not render related categories when categories for add-on do not exist in clientApp', () => {
+      const addon = createInternalAddonWithLang({
+        ...fakeAddon,
+        categories: {
+          [CLIENT_APP_FIREFOX]: ['blogging', 'games'],
```

Comment:
These categories don't exist in `categories` but they are a bit too realistic. Maybe use these categories:

```suggestion
          [CLIENT_APP_FIREFOX]: ['does-not-exist', 'does-not-exist-2'],
```